src/languages/javascript.coffee

```diff
@@ -593,14 +593,22 @@ exports.JavaScriptParser = class JavaScriptParser extends parser.Parser
       when 'CallExpression', 'NewExpression'
         known = @lookupKnownName node
         blockOpts = {}
+        argCount = node.arguments.length
         if known?.fn
           showButtons = known.fn.minArgs? or known.fn.maxArgs?
+          minArgs = known.fn.minArgs or 0
+          maxArgs = if known.fn.maxArgs? then known.fn.maxArgs else Infinity
```

Comment:
I think coffeescript allows us to write this more tersely as `known.fn.maxArgs ? Infinity`